src/main/java/dk/cngroup/lentils/service/ImageService.java

```diff
@@ -45,15 +50,15 @@ public String getDirectory() {
         return newDirectory;
     }
 
-    public void saveImageFile(final HintFormDTO formObject) {
+    public void saveImageFile(final HintFormDTO formObject, final String path) {
         if (!FileUtils.isFilePresentInHintForm(formObject)) {
             return;
         }
         MultipartFile file = formObject.getImage();
-        String pathToSave = getFilePathToSave(file);
-        File dest = new File(pathToSave);
+        File dest = new File(FileUtils.getUserDir() + path);
         try {
             file.transferTo(dest);
+            return;
```

Comment:
Tady by podle me ten return není třeba.

Reply:
Jo, mas pravdu, puvodne jsem vracela String, tak mi to tam zbylo